src/pages/IdeaPannel.astro

```diff
@@ -0,0 +1,42 @@
+---
+import BaseHead from "@astro/base/BaseHead.astro";
+import Header from "@astro/header/Header.astro";
+import Footer from "@astro/footer/Footer.astro";
+import authorConfig from "@config/siteConfig/info.json";
+import { featureFlags } from "@config/featureFlag/featureFlag.json";
+import SpeedInsights from "@vercel/speed-insights/astro";
+import IdeaPannelReact from "@react/IdeasPannel/ideaPannel";
+import { getCollection } from "astro:content";
+const posts = (await getCollection("blog")).sort(
+  (a, b) => a.data.pubDate.valueOf() - b.data.pubDate.valueOf()
+);
+
+try {
+  if (!featureFlags.showIdeaPannel) {
+    return Astro.redirect("/access-denied");
+  }
+} catch (error) {
+  return Astro.redirect("/404");
+}
```

Comment:
In the production there is no need to add console.error, if there is any error then it will redirect to the 404 page and In the analysis Page, I can see which page is causing the issue and then I can use console.error or log to check the error.